### PR TITLE
Grammar check

### PR DESCRIPTION
**Pull request by Shadowtail117**
Ready to go.

### CODE REVIEW

**Comment by Albens** [comment]
@Shadowtail117 24h warning before close.
Gonna take your changes either way, if you can resolve your conflicts.

File: maps/torch/items/summarydocuments.dm [hunk 1]
```diff
@@ -1,130 +1,114 @@
...
-
-/obj/item/document/generic/sol/holocalletiquette
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, containing a small guide on proper use of holocalls from the perspective of the SEV Torch. This one details five points. First, Calls are a one-way interaction. They inform you of the basic situation, you inform them of what you can do to help. Second, Calls are two-way if they request them to be. Third, If an executive decision is required, ask the Senior on the ground. If he does not render a decision, the SEV Torch shall, but with an emphasis on being decisive. Fourth, State aloud when the call connects. It helps cut down on background chatter. Fifth, You don't need to be informed on every specific thing Exploration is doing while on Holocall."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: After some thinking, I decided it's best to leave this one out. There's no way to condense it to make it smaller or more easily readable, and I think making holocall etiquette prescriptive like this isn't that great of a move anyway.

File: maps/torch/items/summarydocuments.dm [hunk 2]
```diff
@@ -1,130 +1,114 @@
...
+	desc = "An \"internal use only\" memo sent from the Commanding Officer to the heads of staff reminding them that bridge officers are qualified in communications and bridge operations only and do not, in fact, double as naval infantry. They should not be deployed like such during minor security incidents, citing numerous reports of broken legs and feet being shot due to mishandling of weapons."
 
 /obj/item/documents/generic/sol/comm
 	name = "ZA-Common reminder"
-	desc = "A memo that has seen better days. It's been crumpled up, dried in coffee and put back up by some vandals. From what's left, it reads: 'Speaking good ZA-Common is a must to be an excellent head of staff. Please exercise moderation with accents.'"
-
-/obj/item/document/generic/sol/holocallloiter
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, requesting people that if a holocall lingers, they should make a point to end it."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: Removing this one as well for similar reasons. That, and it's a bit redundant; I've never seen a holocall drag on unnecessarily long myself.

File: maps/torch/items/summarydocuments.dm [hunk 3]
```diff
@@ -1,130 +1,114 @@
...
-
-/obj/item/documents/generic/sol/officernjp
-	name = "nonjudicial punishments for commissioned personnel"
-	desc = "A \"for official use only\" memorandum detailing options with regards to nonjudicially punishing commissioned uniformed servicemembers. There is no summarized NJP process. For a company-level NJP, a written reprimand/admonition is required. Restriction is capped at thirty days. Field Grade NJPs are the same. General Officer NJPs require a written letter of reprimand or admonition. Punishment includes up to 60 days restriction or 30 days arrest in quarters. Forfeiture of half of one-month�s pay for two months is authorized."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: Tried toying a lot with this to make it as concise as possible but it just didn't work. Plus a lot of the information here directly contradicts the wiki, which should be seen as the primary source of SCUJ.

File: maps/torch/items/summarydocuments.dm [hunk 4]
```diff
@@ -208,160 +192,137 @@
...
-
-/obj/item/document/generic/explo/holocalletiquette
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, containing a small guide on proper use of holocalls from the perspective of the SEV Charon. This one details five points. First, Calls are a one-way interaction. You inform them of the basic situation, they inform you of what they can do to help. Second, Calls are two-way if you request them to be. Third, If an executive decision is required, ask the Senior on the ground. If he does not render a decision, the SEV Torch shall, but with an emphasis on being decisive. Fourth, State aloud when the call connects. It helps cut down on background chatter. Fifth, The SEV Torch doesn't need to be informed on every specific thing Exploration is doing while on Holocall."
-
-/obj/item/document/generic/explo/holocallloiter
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, requesting people that if a holocall lingers, they should make a point to end it."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: See the above review comments for reasons behind these removals.

File: maps/torch/items/summarydocuments.dm [hunk 5]
```diff
@@ -208,160 +192,137 @@
...
+	desc = "Some \"internal use only\" reports discussing the process of dealing with survivors, off-ship personnel, or in-distress civilians. This one details three points that all explorers, Fleet servicemembers, or contractor personnel should attempt to follow when dealing with off-ship persons: First, treat them with the utmost respect at all times, as they are the ones who need aid. Second, do not directly order them around, unless they happen to be getting in the way of SCG objectives. Third, remind them of their rights - they are requiring aid and are to be treated with the best handling, but remind them that they are still on SCG property, and are subject to SCG law at all times."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm:247]: I'm personally still iffy on this one since it seems too prescriptive about handling foreign people, but it seems alright as-is.

File: maps/torch/items/summarydocuments.dm [hunk 6]
```diff
@@ -208,160 +192,137 @@
...
-	desc = "It's a portable map of the Petrov. Someone wrote over it in big letters: \"What the fuck is the Petrov?\""
+	desc = "It's a portable map of the Petrov. Someone wrote over it in big letters: \"What the fuck is the Petrov?\", and it's now unusable."
 
 /obj/item/documents/generic/sci/fuckingdoors
 	name = "windoor guide"
-	desc = "An \"internal use only\" visual guide for the proper operation of xenobiology containment. Features a hand-written note saying \"PAY ATTENTION TO THIS!\" in large letters."
-
-/obj/item/documents/generic/corp/bodyarmourbad
-	name = "LPA recommendations"
-	desc = "Some \"internal use only\" corporate documents, detailing that having any loss prevention associate wear body armor and openly carry a weapon in secure areas"
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: LPAs were removed

File: maps/torch/items/summarydocuments.dm [hunk 7]
```diff
@@ -208,160 +192,137 @@
...
+	desc = "An \"internal use only\" visual guide for the proper operation of xenobiology containment doors. It features a hand-written note saying \"PAY ATTENTION TO THIS!\" in large letters."
 
 /obj/item/documents/generic/corp/coffeecut
 	name = "corporate spending records"
-	desc = "A \"confidential\" memo on the current usage of funds provided to SEV Torch Corporate Liaisons, apparently there's threats of a strike over a cut to the coffee budget by head office"
-
-/obj/item/documents/generic/corp/LPArenting
-	name = "LPA usage notice"
-	desc = "A \"confidential\" notice reminding staff aboard the SEV Torch that their security staff are for their use only, and renting them out as extra manpower to security is a violation of their contract"
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: LPAs were removed

File: maps/torch/items/summarydocuments.dm [hunk 8]
```diff
@@ -208,160 +192,137 @@
...
 
 /obj/item/documents/generic/corp/glueandtape
 	name = "proper document disposal"
 	desc = "Some \"internal use only\" corporate documents detailing the proper ways to dispose of top secret and classified documents. It highlights that burning is most applicable, and shows pictures of shredded documents reconstructed with glue and sticky tape with a red \"Fail\" written below."
 
-/obj/item/documents/generic/corp/lpainfo
-	name = "LPA reminder"
-	desc = "An \"internal use only\" corporate document informing company Liasions of the Loss-Prevention Agents job aboard the ship. It cites concerns about the misuse of the LPA and general mistrust towards them from crew, and how their actions damages the Company if they keep acting in this manner. It reminds them that they are from an established industry, and should have experience in many aspects other than carrying a firearm. It specifies good observation skills, solid restraint, an understanding of escalation rules in regards to his position, and that their experience as an industry professional can be useful as an advisor to his tasked Liasion."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: LPAs were removed

File: maps/torch/items/summarydocuments.dm [hunk 9]
```diff
@@ -208,160 +192,137 @@
...
-
-/obj/item/documents/generic/sec/article31
-	name = "SCUJ rights"
-	desc = "An \"unclassified\" memorandum detailing a uniformed servicemember's SCUJ rights. It specifies that no person subject to this chapter may interrogate or request any statement from an accused or a person suspected of an offense without first informing him of the nature of the accusation and advising him that he does not have to make any statement regarding the offense of which he is accused or suspected, and that any statement made by him may be used as evidence against him in a trial by court-martial. No statement obtained from any person in violation of this article, or through the use of coercion, unlawful influence, or unlawful inducement, may be received in evidence against them in a trial by court-martial."
```
Comment (Shadowtail117) [inline maps/torch/items/summarydocuments.dm]: Removed as per discord conversation

**Review by Shadowtail117 — COMMENTED** [review]
A few of these reports I also changed the meaning as well as the wording, while trying to maintain the same intent. Hope you enjoy!

File: maps/torch/items/summarydocuments.dm (hunk 6 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: https://gyazo.com/be6fd8d53a5c2745d2f572416f4249f2

File: maps/torch/items/summarydocuments.dm (hunk 7 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: Still in game as per above screenshot.

File: maps/torch/items/summarydocuments.dm (hunk 8 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: Still in game as per above screenshot.

**Comment by Shadowtail117** [comment]
Merge conflicts fixed....I think?

**Comment by Shadowtail117** [comment]
Okay nevermind, ended up overwriting a few reports that were added after I made this branch. Hold on while I add them back

File: maps/torch/items/summarydocuments.dm (hunk 1 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: Agreed.
A shame, but I'm unable to come up with ways to condense it.

File: maps/torch/items/summarydocuments.dm (hunk 2 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: Agreed as above.

File: maps/torch/items/summarydocuments.dm (hunk 3 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm]: Eckles ruled directly (iirc) that this was not going to get in anyway, unless it was as per the wiki.
In which case, it would just be a wiki doc pasted directly, which is dumb.

Agreed.

File: maps/torch/items/summarydocuments.dm (hunk 5 above)
Comment (Albens) [reply maps/torch/items/summarydocuments.dm:247]: Bit too wordy, but workable.

**Comment by Shadowtail117** [comment]
I am about at my wit's end with this

See https://github.com/Albens/Baystation12/pull/6
